api/config/v2alpha1/projectconfig_types.go

```diff
@@ -37,6 +37,10 @@ type ProjectConfig struct {
 	// running multiple controllers in the same cluster.
 	ControllerClass string `json:"controllerClass"`
 
+	// DeletionProtectionDefault sets the default to with regards to deletion
```

Comment:
This part "default to with regards to deletion" seems to be broken.

Reply:
Also, in commit message you made a typo and typed "my default" instead of "by default"